200_account/300_patch_me.md

```diff
@@ -4,6 +4,11 @@
 
 ## `PATCH`{.verb} `/auth/me/`
 
+<div class='info-yellow'>
+<b>Rate Limits:</b><br>
+Email address updates are limited to one attempt every month.
+</div>
+
```

Comment:
Arguably, ever merging this 10%-done endpoint into master was a miss on my part. I'm still not a 100% on whether it'll stay or whether I'll replace login with "sign in with google". Either way, probably no point in documenting it, since it doesn't even work :sweat_smile: